### ISSUE TITLE
[PROPOSAL] The Golden Land (How to save the series of Golden Items?)

### ISSUE THREAD

**Issue by Hey4rthur** [sunflower-land/sunflower-land#968]
Hello **everyone**, I hope you are well!

I bring ideas for **future Golden Items,** I feel a strong appreciation for the **Golden Cauliflower,** and unfortunately the **tendency of the item is to devalue itself in the face of halvings,** after all the item will lose its usefulness, so I've been thinking about some features that could be interesting in the **series of golden items.** (I make it clear that I'm not just referring to the Golden Cauliflower, but also the upcoming Golden Items to come)

As we all know **Gold** is seen in the world as a sign of lust, and based on that it would be interesting if these same items also better represent this idea within the farm.

### **_Benefits of owning a Golden Item:_**
_*Golden Land
*Main House becomes more luxurious_

### **_If the Gold Item is a Plant:_**
_*Double in Collection **(as Golden Cauliflower already is)**
*Specific Plant Seeds Free or half price.
*Every halving you receive a 30% more buff on the sale of the Plant in Specific **(This would help not to end the functionality of the Item)**
*And when you collect the specific plant you have a 0.03% chance to drop 1 Gold_

### **_And now applying these same Buffs on the Golden Cauliflower would be:_**
_*Golden Land
*Main House becomes more luxurious
*Double on Cauliflower Collection
*Cauliflower seeds free or at half price
*30% more on Cauliflower sales
*And each cauliflower you collect you have a 0.03% chance to drop 1 Gold_

**Golden Lands** are more likely to find **rare ores** _(Diamond, Ruby, Emerald)_ in land expansions (I suggest Double the chance, for example if the chance of finding these ores is 1% then it goes up to 2%)

I believe that these small changes are certainly interesting and they would avoid the death of some NFT's, and because the items are scarce, I believe it would not affect the game's economy so much, Anyway these are ideas that can certainly be improved and matured, I'm totally willing to listen and even analyze your constructive criticism, it will be a pleasure to hear what you have to say!!

**_(If you found this at least interesting, react to this post to help achieve DEVS)_**

![GOLDEN LAND2](https://user-images.githubusercontent.com/104866298/169415489-f0e3245c-0004-42e7-9ec4-551393832eee.png)

**Comment by ZikOosWF** [comment]
Oh What an ideas...I like it ! 😍🔥

**Comment by Telknub** [comment]
I like this idea.
I think this can be expanded even more.

Maybe every expensive NFT changes your land to a different type with special buffs to your land depending on the NFT, and if you have more than one of these NFTs you have to choose what buff you would get, to make everything balanced.

A few examples: 
The golden land (golden cauliflower): gives a small chance to drop gold when harvesting cauliflowers (maybe 1/day average), change your house to a golden one (cosmetic) and also when you expand your land you get better odds for rare materials and gold.

Mysterious land (mysterious parsnip): this one could increase all mutants odds (more than carrot sword), change the house to a mysterious one and also gives better odds to find rare materials and gold on land expansions.

Scary land (kuebiko): this land would make everything a bit cheaper (land expansions, animals, tools and maybe even NFTs?) by a small % (10-20% maybe) and also change the house to a scary one. 

The houses would be an optional cosmetic and you would change the buff and the house skin (if you have more than one of these NFTs) by clicking on a menu on the house. To prevent abuse these buffs could also have a cooldown to change beetween them, and if you withdraw the NFT you lose the buff, but the cooldown to change would remain the same.

**Comment by brian-man** [comment]
IMO it's a bit "reward people that already own it" and a bit "rugpull" for those that actually prefer the existing effects of the items to change them after they are crafted and the supply is no longer available for crafting.

I would prefer that new FTs be introduced for such significantly different behavior rather than completely change already crafted and/or purchased ones.

**Comment by vcferro** [comment]
I loved the idea, as devs already said we are in beta and NFTs should be buffed or nerfed... Due to the rarity and price of this item, changes are necessary to reach the above-mentioned floor price of 3 eth. it is necessary for this item to gain a buff otherwise future nfts will be devalued... who will risk making new nfts when they know they won't be worth anything?

**Comment by brian-man** [comment]
> Due to the rarity and price of this item, changes are necessary to reach the above-mentioned floor price of 3 eth.

Huh?

**Comment by Telknub** [comment]
> IMO it's a bit "reward people that already own it" and a bit "rugpull" for those that actually prefer the existing effects of the items to change them after they are crafted and the supply is no longer available for crafting.
> 
> I would prefer that new FTs be introduced for such significantly different behavior rather than completely change already crafted and/or purchased ones.

In my opinion the problem with these NFTs is that their buff will quicly decrease an be almost worthless after 1 or 2 halvings, with a difference of less than 2 SFL/day.

These NFTs were very strong when they were designed but since their creation we got 3 halvings and now their buffs are not really strong for the price and supply they have.

This is a good way to retain value on the NFTs that are supposed to be the most valuables NFTs in the game. Future expensive and low supply NFTs would also have buffs like these, and these buffs are not even that strong since these NFTs will still lose their power in SFL generation.

**Comment by brian-man** [comment]
> > IMO it's a bit "reward people that already own it" and a bit "rugpull" for those that actually prefer the existing effects of the items to change them after they are crafted and the supply is no longer available for crafting.
> > I would prefer that new FTs be introduced for such significantly different behavior rather than completely change already crafted and/or purchased ones.
> 
> In my opinion the problem with these NFTs is that their buff will quicly decrease an be almost worthless after 1 or 2 halvings, with a difference of less than 2 SFL/day.
> 
> These NFTs were very strong when they were designed but since their creation we got 3 halvings and now their buffs are not really strong for the price and supply they have.
> 
> This is a good way to retain value on the NFTs that are supposed to be the most valuables NFTs in the game. Future expensive and low supply NFTs would also have buffs like these, and these buffs are not even that strong since these NFTs will still lose their power in SFL generation.

You probably want to read the Land Expansion proposal.  GC will be stronger with land expansion.

**Comment by Telknub** [comment]
> > Due to the rarity and price of this item, changes are necessary to reach the above-mentioned floor price of 3 eth.
> 
> Huh? It kind of sounds like you're just saying "I got lucky and crafted 100 of the 1000 contested GC and want to be paid 3 eth".

I think he is referencing an comment Adam made on a issue that the tier 3 and golden NFTs would have a floor price at about 3 eth.

**Comment by brian-man** [comment]
> > > Due to the rarity and price of this item, changes are necessary to reach the above-mentioned floor price of 3 eth.
> > 
> > 
> > Huh? It kind of sounds like you're just saying "I got lucky and crafted 100 of the 1000 contested GC and want to be paid 3 eth".
> 
> I think he is referencing an comment Adam made on a issue that the tier 3 and golden NFTs would have a floor price at about 3 eth.

Well they sold for 3.5 ETH and 1.1 ETH from the dev supply before they were even craftable, so not sure what relevance that has to anything anymore. :|

**Comment by brian-man** [comment]
> This is a good way to retain value on the NFTs that are supposed to be the most valuables NFTs in the game.

Completely changing how an FT works after it's crafted sets a horrible precedent for the value of everything in the game.

**Comment by Telknub** [comment]
> > > IMO it's a bit "reward people that already own it" and a bit "rugpull" for those that actually prefer the existing effects of the items to change them after they are crafted and the supply is no longer available for crafting.
> > > I would prefer that new FTs be introduced for such significantly different behavior rather than completely change already crafted and/or purchased ones.
> > 
> > 
> > In my opinion the problem with these NFTs is that their buff will quicly decrease an be almost worthless after 1 or 2 halvings, with a difference of less than 2 SFL/day.
> > These NFTs were very strong when they were designed but since their creation we got 3 halvings and now their buffs are not really strong for the price and supply they have.
> > This is a good way to retain value on the NFTs that are supposed to be the most valuables NFTs in the game. Future expensive and low supply NFTs would also have buffs like these, and these buffs are not even that strong since these NFTs will still lose their power in SFL generation.
> 
> You probably want to read the Land Expansion proposal. GC will be stronger with land expansion.

I already read it. 
Considering these NFTs get buffed now,  they can be nerfed later if they get too strong...

I don't think any rare NFT owner would get mad if their NFT is nerfed at the same time a big update that would make that NFT a lot stronger is dropped. This is considering that everything is explained before happening.

**Comment by brian-man** [comment]
> Considering these NFTs get buffed now, they can be nerfed later if they get too strong...

I think you misunderstood my point.  GC is already powerful, and will be crazy powerful with land expansion.  Instead of fundamentally changing something that already exists just wait until land expansion.

**Comment by Telknub** [comment]
> > > > Due to the rarity and price of this item, changes are necessary to reach the above-mentioned floor price of 3 eth.
> > > 
> > > 
> > > Huh? It kind of sounds like you're just saying "I got lucky and crafted 100 of the 1000 contested GC and want to be paid 3 eth".
> > 
> > 
> > I think he is referencing an comment Adam made on a issue that the tier 3 and golden NFTs would have a floor price at about 3 eth.
> 
> Well they sold for 3.5 ETH and 1.1 ETH from the dev supply before they were even craftable, so not sure what relevance that has to anything anymore. :|
> 
> Edit: This quote?
> 
> > To be fair, a T3 item is extremely rare and already has a floor price of 1 ETH.
> > Apr 19
> > [**https://github.com/sunflower-land/sunflower-land/issues/574**](https://github.com/sunflower-land/sunflower-land/issues/574#issuecomment-1103291430)
> 
> This was before crafting opened _and_ it wasn't a statement of goals but a statement of current status. Weird what people read into things. Also the comment says 1 ETH not 3.
> 
> It's also weird he called it a floor price when there were only 2 for sale. If anything I would call that a ceiling, not a floor.

https://github.com/sunflower-land/sunflower-land/issues/574#issuecomment-1105895828

I was talking about this one

**Comment by brian-man** [comment]
> [#574 (comment)](https://github.com/sunflower-land/sunflower-land/issues/574#issuecomment-1105895828)
> 
> I was talking about this one

>> Tier 3 items .... if you want something at a floor price of 3 eth, you can't expect it to get it for free without giving back to the game's economy (i.e. buying resources on a exchange)

So you're saying they should buff Foreman Beaver and Kuebiko since they are trading under 1 ETH?

What does that have to do with Golden Cauliflower, which is not a T3?

**Comment by vcferro** [comment]
how is the golden cauli strong if you can get almost the same results planting carrots with a free rabbit nft?

> > Considering these NFTs get buffed now, they can be nerfed later if they get too strong...
> 
> I think you misunderstood my point. GC is already powerful, and will be crazy powerful with land expansion. Instead of fundamentally changing something that already exists just wait until land expansion.
> 
> Also nothing "changed" that made GC different from what was advertised (in a bad way). The halvings were known, not even new to v2. The behavior of the NFT was known. Just because some people didn't understand the mechanics of the game or the item doesn't mean anybody or anything "owes" them a rework. Insanity.

**Comment by Telknub** [comment]
> > Considering these NFTs get buffed now, they can be nerfed later if they get too strong...
> 
> I think you misunderstood my point. GC is already powerful, and will be crazy powerful with land expansion. Instead of fundamentally changing something that already exists just wait until land expansion.
> 
> Also nothing "changed" that made GC different from what was advertised (in a bad way). The halvings were known, not even new to v2. The behavior of the NFT was known. Just because some people didn't understand the mechanics of the game or the item doesn't mean anybody or anything "owes" them a rework. Insanity.

So they have to nerf other NFTs, because the difference of someone with the almost free rabbit to someone with the rare and expensive GC is only 2 tokens/day. The main reason to make these expensive NFTs is to have something in return that actually makes sense of their price.  

Its a lot easier for someone to make 5 accounts for their family teach them how to play and farm 4 times more than it was to get a single GC....

**Comment by brian-man** [comment]
> how is the golden cauli strong if you can get almost the same results planting carrots with a free rabbit nft?

Show your math?  Also, Easter Bunny was not free.

**Comment by Telknub** [comment]
> > [#574 (comment)](https://github.com/sunflower-land/sunflower-land/issues/574#issuecomment-1105895828)
> > I was talking about this one
> 
> > > Tier 3 items .... if you want something at a floor price of 3 eth, you can't expect it to get it for free without giving back to the game's economy (i.e. buying resources on a exchange)
> 
> So you're saying they should buff Foreman Beaver and Kuebiko since they are trading under 1 ETH?
> 
> What does that have to do with Golden Cauliflower, which is not a T3?

If you read my first comment, my idea is to buff kuebiko, GC and MP.
Beaver is not included because he does not lose any value when a halving happens.

**Comment by brian-man** [comment]
> Its a lot easier for someone to make 5 accounts for their family teach them how to play and farm 4 times more than it was to get a single GC....

So either you're against families or you're saying the multi-accounting efforts aren't hardcore enough.  I'm not sure which.

Not sure why you single out GC for that concern.  It affects the entire game.

**Comment by Telknub** [comment]
> > how is the golden cauli strong if you can get almost the same results planting carrots with a free rabbit nft?
> 
> Show your math? Also, Easter Bunny was not free.

**Almost free** 
You can use the NFT army table and see for yourself that a rabbit makes almost 7 tokens/day and the gc almost 10

**Comment by brian-man** [comment]
> If you read my first comment, my idea is to buff kuebiko, GC and MP.
> Beaver is not included because he does not lose any value when a halving happens.

It sounds like that should be a separate proposal.  Nothing about Kuebiko or MP sounds "golden" to me.

**Comment by brian-man** [comment]
> **Almost free**

Sunflower Seeds are "almost free" too.  Should we consider them overpowered?

> You can use the NFT army table and see for yourself that a rabbit makes almost 7 tokens/day and the gc almost 10

"NFT army table"?

**Comment by Telknub** [comment]
> > Its a lot easier for someone to make 5 accounts for their family teach them how to play and farm 4 times more than it was to get a single GC....
> 
> So either you're against families or you're saying the multi-accounting efforts aren't hardcore enough. I'm not sure which.
> 
> Not sure why you single out GC for that concern. It affects the entire game.

The original price of the golden cauli was like 10 gold and 100 cauliflowers
They changed it because it was too strong for that price, now it is to weak for the price everyone paid for...

**Comment by Telknub** [comment]
> > If you read my first comment, my idea is to buff kuebiko, GC and MP.
> > Beaver is not included because he does not lose any value when a halving happens.
> 
> It sounds like that should be a separate proposal. Nothing about Kuebiko or MP sounds "golden" to me.

Did you read my first comment on this issue?

**Comment by brian-man** [comment]
> The original price of the golden cauli was like 10 gold and 100 cauliflowers
> They changed it because it was too strong for that price, now it is to weak for the price everyone paid for...

And you were forced to buy it?  If the value prop wasn't there, why did you buy it?

**Comment by brian-man** [comment]
> Did you read my first comment on this issue?

Yes. I found it all over the map and frankly not viable to basically restructure the entire FT market because you have buyer's remores.

**Comment by Telknub** [comment]
> > The original price of the golden cauli was like 10 gold and 100 cauliflowers
> > They changed it because it was too strong for that price, now it is to weak for the price everyone paid for...
> 
> And you were forced to buy it? If the value prop wasn't there, why did you buy it?
> 
> There were 900 people behind you ready to buy it. You didn't have to.

I bought it because I am on this beta since the beggining and I started farming gold for it like two months ago.



> > Did you read my first comment on this issue?
> 
> Yes. I found it all over the map and frankly not viable to basically restructure the entire FT market because you have buyer's remorse.
> 
> They didn't change the item after you bought it, so there's no "bait and switch" yet you're asking them to do that to everyone else because you're not happy with what you bought.

The game is still in beta so things can still change. In the future when they release other similar NFTs other people will see that they devalue and are not worth, like it is happening with the parsnip that is taking days to run out of supply.

Also why are you getting so agressive? I am just showing my reasoning behind the ideas.

I think it is a lot better to have lots of rare and OP NFTs to make multiaccounting less valuable. Someone who pays a lot for an NFT would recieve something actually worth in exchange.

Imo if these NFTs continue like this there will be no reason for new players that actually want to invest a lot in rare and expensive NFTs to do it. It will be just a normal crafting and farming game with limited supply where the only things that sell for profit in the market are resources.....

If the expensive stuff that is supposed to have a high value is not selling, the game will only reward those who make multiaccounting and bots to farm a lot.

Yes, I know the devs are trying to diminish as much as possible bots and multiaccounting, but the current system rewards them a lot more than the people actually invested in the game.

**Comment by brian-man** [comment]
> Also why are you getting so agressive? I am just showing my reasoning behind the ideas.

Aggressive? No. Just direct. When the premise is "changing everything about the FTs in the game" _after_ people have already invested in them, I think that's insane.

**Comment by Hey4rthur** [comment]
I found it quite aggressive, but I ask you, do you think the value of Golden Cauliflower makes sense? (it's cheaper than the gold egg
![0-0](https://user-images.githubusercontent.com/104866298/169429298-f1285b71-a1be-4f40-ab79-30e58af0f672.png)
)

**Comment by brian-man** [comment]
> I found it quite aggressive, but I ask you, do you think the value of Golden Cauliflower makes sense? (it's cheaper than the gold egg)
> 
> ![0-0](https://user-images.githubusercontent.com/104866298/169429298-f1285b71-a1be-4f40-ab79-30e58af0f672.png)

The market is the market.

I think people are stupid for selling their GCs so cheap (which is part of why I'm likely buying one this week).

Gold Egg is a wildcard with unknown value that people love to speculate on and/or try to money grab from.  Recently (Chickens proposal) there's been some light shed on what they might be used for.  I expect the market price to go up or down significantly (not sure which) as sentiment evolves based on that proposal.

**Comment by Telknub** [comment]
> > Also why are you getting so agressive? I am just showing my reasoning behind the ideas.
> 
> Aggressive? No. Just direct. When the premise is "changing everything about the FTs in the game" _after_ people have already invested in them, I think that's insane.
> 
> > I think it is a lot better to have lots of rare and OP NFTs
> 
> Sure. Add new FTs. That's what I said earlier.
> 
> Completely changing existing ones is what I'm pushing back on.
> 
> > If the expensive stuff that is supposed to have a high value is not selling, the game will only reward those who make multiaccounting and bots to farm a lot.
> > Yes, I know the devs are trying to diminish as much as possible bots and multiaccounting, but the current system rewards them a lot more than the people actually invested in the game.
> 
> Rewarding them by blocking their accounts from playing?

You know that it takes time to ban some farms right? Also not every system is perfect and there are limits where someone could possibly abuse without being detected.

Right now the buffs those expensive NFTs have are not enough imo if you put in compairson to the rabbit that a lot of people got for a very low price and now can be bought at $5 on OS.

If they add NFTs in the future that are stronger than the gc/kuebiko/mp wouldn't it also be unfair to everyone who was in the game in the beggining and tried to make one of the first expensive NFTs?

**Comment by brian-man** [comment]
> Right now the buffs those expensive NFTs have are not enough imo if you put in compairson to the rabbit that a lot of people got for a very low price and now can be bought at $5 on OS.

If the rabbit is so good and the GC is so bad, why don't you sell your GC and buy 80 rabbits?  And then sell the rabbits in November when people wake up?

**Comment by brian-man** [comment]
> If they add NFTs in the future that are stronger than the gc/kuebiko/mp wouldn't it also be unfair to everyone who was in the game in the beggining and tried to make one of the first expensive NFTs?

No.  Nobody said anything about "permanently the best ever" about anything in this game and they never should.  If they do, it means the game is closing down soon.

**Comment by Telknub** [comment]
> Aggressive? No. Just direct. 



> > Right now the buffs those expensive NFTs have are not enough imo if you put in compairson to the rabbit that a lot of people got for a very low price and now can be bought at $5 on OS.
> 
> If the rabbit is so good and the GC is so bad, why don't you sell your GC and buy 80 rabbits? And then sell the rabbits in November when people wake up?

Are you sure you are not being agressive?

**Comment by brian-man** [comment]
> Are you sure you are not being agressive?

Pointless discussion.  How about focusing on the game?

**Comment by Telknub** [comment]
> > Are you sure you are not being agressive?
> 
> Pointless discussion. How about focusing on the game?
> 
> If I was, I would use words like "twat" and "degen" but I don't.

Pointless is you saying for me to buy 80 rabbits instead of countering my reasoning on why they should buff the NFTs.



> > If they add NFTs in the future that are stronger than the gc/kuebiko/mp wouldn't it also be unfair to everyone who was in the game in the beggining and tried to make one of the first expensive NFTs?
> 
> No. Nobody said anything about "permanently the best ever" about anything in this game and they never should. If they do, it means the game is closing down soon.
> 
> Further, every day you own a buff and are benefitting from it you are extracting value. At some point you should break even and if they removed the item from the game, 1 day later you're ahead.
> 
> Don't assume you anything you ever buy will never be obsolete. That's just being naive and setting yourself up for disappointment.
> 
> Steve has said at least 4 times that future NFTs (he uses the term wrong) will be better than current ones.

I also did not say at any point that the gc is supposed to be the best NFT forever, just that it is supposed to be one of the best NFTs... 

Also where did Steve said this? You know Steve is not a part of the core team, right? He is just the marketing, PR and community manager guy...

**Comment by brian-man** [comment]
> I also did not say at any point that the gc is supposed to be the best NFT forever, just that it is supposed to be one of the best NFTs...

It is one of the best current FTs.  You just don't want to believe it is, for some reason.

> You know Steve is not a part of the core team, right? He is just the marketing, PR and community manager guy...

You're welcome to think that.  But the upcoming event that significantly impact Land Expansion only exists because Steve is involved.  I'm far more worried about the impact of that than any item that currently exists in the game.

**Comment by Telknub** [comment]
> > I also did not say at any point that the gc is supposed to be the best NFT forever, just that it is supposed to be one of the best NFTs...
> 
> It is one of the best current FTs. You just don't want to believe it is, for some reason.
> 
> > You know Steve is not a part of the core team, right? He is just the marketing, PR and community manager guy...
> 
> You're welcome to think that. But the upcoming event that significantly impacts Land Expansion only exists because Steve is involved. I'm far more worried about the impact of that than any item that currently exists in the game.

Yes it is one of the best NFTs. But the buff is small if you compare to its price. The time it takes for you to break even is too long. Adam said he would tweak the price for it to have a 20-30 day return in the beggining of the beta because it was too op (2 days return). Now it is currently 120 days return and it will double next halving.

Where did you see the upcoming event will have a impact on land expansion? Land expansion will only come in like 3-4 months and this event will be released soon I think.....

**Comment by brian-man** [comment]
> Where did you see the upcoming event will have a impact on land expansion? Land expansion will only come in like 3-4 months and this event will be released soon I think.....

Listen to the Steve vimeo from yesterday.  Specifically around 1hr 10min.

**Comment by ElcomJ** [comment]
Hey brian, you really like stevee right?

**Comment by brian-man** [comment]
> Hey brian, you really like stevee right?

Weird question, but sure.

**Comment by ll3x** [comment]
I think the question is not about "changing the NFT", but that the value in relation to the development of the game does not change, we are all aware of the halving, so why not collect sfl from the beginning to the end of the game, when the value of the coin will be at the peak? it's all about the game itself, you play the game and expect appropriate buffs from certain items, to be honest, I don't like all the discussions about buffs or any changes, because it introduces an imbalance in the game, it looks like a scam. I've written about this in many places and tried to draw attention to the problem with scarecrows, in particular t3, and I also wrote a rather elegant solution to these issues, we just need to add mutants to the game ...
and also these mutant cultures will change to always have a decent reward before halving, look, now we can produce cakes at a stable price and stable cost, do you think this is true in relation to the t3 scarecrow, the cost and quantity of which highlights him against the background of all the others? After 2-3 halvings, selling a cake will get you 5-10-20 sfl, with a seed price of 0.00001 sfl, this is no longer fair in relation to growing seeds.
Yes, we will develop, yes, maybe there will be lands, but people must have some kind of stability, NFT is the best resource of stability, since their supply is limited, especially the rarest ones. so why not make these NFTs really worthwhile without changing their original characteristics, but adding some content to them, especially since you don’t need to invent anything, the mutants are already there, you just need to sell them ... add cauliflower mutant seeds that will be consistently sold for 0.3-0.5 cfl, and which will cost a stable 0.05 cfl and over time, buying mutant seeds will become very expensive , but they will bring a standard income.
Bottom line: GC owners are happy, T3 scarecrow owners are happy, carrot sword owners are the most satisfied =)
